data_generation/utils/chatgpt_prompt_utils.py

```diff
@@ -0,0 +1,91 @@
+import ast
+
+import openai
+from openai.error import ServiceUnavailableError
+
+
+def get_chatgpt_response(content, model="gpt-3.5-turbo"):
+
+    completion = openai.ChatCompletion.create(
+        model=model,
+        messages=[
+            {
+                "role": "user",
+                "content": content,
+            }
+        ],
+    )
+    gpt_response = completion.choices[0].message.content
```

Comment:
can we add error handling code here? For example we might not be connected to the internet